### ISSUE TITLE
GunItemHook doesn't get shot even triggered by sentries

### ISSUE THREAD

**Issue by DerpyHoowes** [SmartlyDressedGames/Unturned-3.x-Community#4379]
Guns with GunItemHook don't get their onShotFired Event triggered by sentries that are using them.

**Comment by SDGNelson** [comment]
You're right that none of that logic was copied over to sentries. Bah, the duplication of code between the player guns and sentry guns is very annoying.
